Relativity.Transfer.Sample/SampleHelpers/AsperaSampleHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿// // ----------------------------------------------------------------------------
+// // <copyright file="AsperaSampleHelper.cs" company="kCura Corp">
+// //   kCura Corp (C) 2021
+// // </copyright>
+// // ----------------------------------------------------------------------------
+
+namespace Relativity.Transfer.Sample.SampleHelpers
+{
+    public class AsperaSampleHelper : AbstractSampleHelper
+    {
+        public override string TransferModeName { get; } = "Aspera";

```

Comment:
Maybe `TransferMode.Aspera.ToString()` could be used instead?

Reply:
Done